sapp/ui/trace.py

```diff
@@ -103,6 +120,25 @@ def from_record(
     def is_leaf(self) -> bool:
         return self.callee_port in LEAF_NAMES
 
+    def get_clean_caller(self, tool: str) -> str:
+        return self._clean_callable_name(self.caller, tool)
+
+    def get_clean_callee(self, tool: str) -> str:
+        return self._clean_callable_name(self.callee, tool)
+
+    def _clean_callable_name(self, name: str, tool: str, depth: int = 1) -> str:
+        if name in LEAF_NAMES:
+            return name
+        if tool == "mariana-trench":
+            package_and_class, method_name = name.split(".")
+            method_name = method_name.split(":")[0]  # parse
+            package_and_class = package_and_class.strip(";").split("/")[
+                depth * -1
+            ]  # parse Lpackage1/package2/class;.
```

Comment:
ditto, comment doesn't really help. Maybe a useful comment would be an example of the full name at the top of this code.